client/src/commands.ts

```diff
@@ -211,23 +211,24 @@ export function startLanguageServer(
   };
 
   function getV8Flags() {
-    const envFlags = process.env.DENO_V8_FLAGS ?? "";
-    const hasMemoryFlagInEnv = envFlags.includes("--max-old-space-size") ??
-      false;
+    let v8Flags = process.env.DENO_V8_FLAGS ?? "";
     if (
-      hasMemoryFlagInEnv &&
+      v8Flags.includes("--max-old-space-size") &&
```

Comment:
Check for `--max_old_space_size` too

Reply:
Added. They should have done one or the other.